apps/src/code-studio/progressRedux.ts

```diff
@@ -415,17 +450,15 @@ function sendReportHelper(
   // The server does not appear to use the user ID parameter,
   // so just pass 0, like some other milestone posts do.
   const userId = 0;
+  extraData = extraData || {};
```

Comment:
Is it possible for a program property with an empty or falsey value in the data object sent to the server when it's not desired? Given the way `sendReportHelper` is used, it looks like not?

Reply:
It shouldn't be possible (if we use typescript everywhere we call this function, which we currently do 😅). We also only send a report right now with a program for predict levels, and we only allow submitting a response when you have some response filled in.